src/ir_Mitsubishi.h

```diff
@@ -9,10 +9,10 @@
 //   Brand: Mitsubishi,  Model: HC3000 Projector
 //   Brand: Mitsubishi,  Model: MS-GK24VA A/C
 //   Brand: Mitsubishi,  Model: KM14A 0179213 remote
-//   Brand: Mitsubishi Electric,  Model: PEAD-RP71JAA Ducted A/C
-//   Brand: Mitsubishi Electric,  Model: 001CP T7WE10714 remote
-//   Brand: Mitsubishi Electric,  Model: MSH-A24WV / MUH-A24WV A/C
-//   Brand: Mitsubishi Electric,  Model: KPOA remote
+//   Brand: Mitsubishi Electric,  Model: PEAD-RP71JAA Ducted A/C (136bit)
+//   Brand: Mitsubishi Electric,  Model: 001CP T7WE10714 remote (136bit)
+//   Brand: Mitsubishi Electric,  Model: MSH-A24WV / MUH-A24WV A/C (112bit)
+//   Brand: Mitsubishi Electric,  Model: KPOA remote (112bit)
```

Comment:
Ah, you are doing it. My bad.
Please use the `PROTOCOL_NAME` instead. It's more unique.

Reply:
```
//   Brand: Mitsubishi Electric,  Model: MSH-A24WV / MUH-A24WV A/C (MITSUBISHI112)
```
Will be 83 columns, split to 2 lines, one model on each?

Reply:
Sounds like a plan.